src/dashboard/librarysettings.js

```diff
@@ -2,45 +2,47 @@ define(["jQuery", "loading", "libraryMenu", "fnchecked", "emby-checkbox", "emby-
+        $("#chkSaveMetadataHidden", page).checked(config.SaveMetadataHidden);
+        loading.hide();
     }
 
     function loadMetadataConfig(page, config) {
-        $("#selectDateAdded", page).val(config.UseFileCreationTimeForDateAdded ? "1" : "0")
+        $("#selectDateAdded", page).val(config.UseFileCreationTimeForDateAdded ? "1" : "0");
     }
 
     function loadFanartConfig(page, config) {
-        $("#txtFanartApiKey", page).val(config.UserApiKey || "")
+        $("#txtFanartApiKey", page).val(config.UserApiKey || "");
     }
 
     function saveFanart(form) {
         ApiClient.getNamedConfiguration("fanart").then(function(config) {
-            config.UserApiKey = $("#txtFanartApiKey", form).val(), ApiClient.updateNamedConfiguration("fanart", config)
-        })
+            config.UserApiKey = $("#txtFanartApiKey", form).val(), ApiClient.updateNamedConfiguration("fanart", config);
+        });
     }
 
     function saveMetadata(form) {
         ApiClient.getNamedConfiguration("metadata").then(function(config) {
-            config.UseFileCreationTimeForDateAdded = "1" === $("#selectDateAdded", form).val(), ApiClient.updateNamedConfiguration("metadata", config)
+            config.UseFileCreationTimeForDateAdded = "1" === $("#selectDateAdded", form).val(), ApiClient.updateNamedConfiguration("metadata", config);
         })
     }
 
     function alertText(options) {
         require(["alert"], function(alert) {
-            alert(options)
-        })
-    }
-
-    function onSubmitFail(response) {
-        loading.hide(), response && 404 === response.status ? alertText("The metadata path entered could not be found. Please ensure the path is valid and try again.") : response && 500 === response.status && alertText("The metadata path entered is not valid. Please ensure the path exists and that Jellyfin server has write access to the folder.")
+            alert(options);
+        });
     }
 
     function onSubmit() {
         loading.show();
         var form = this;
-        return ApiClient.getServerConfiguration().then(function(config) {
-            config.SaveMetadataHidden = $("#chkSaveMetadataHidden", form).checked(), config.EnableTvDbUpdates = $("#chkEnableTvdbUpdates", form).checked(), config.EnableTmdbUpdates = $("#chkEnableTmdbUpdates", form).checked(), config.EnableFanArtUpdates = $("#chkEnableFanartUpdates", form).checked(), config.MetadataPath = $("#txtMetadataPath", form).val(), config.MetadataNetworkPath = $("#txtMetadataNetworkPath", form).val(), config.FanartApiKey = $("#txtFanartApiKey", form).val(), ApiClient.updateServerConfiguration(config).then(Dashboard.processServerConfigurationUpdateResult, onSubmitFail)
```

Comment:
why did you remove `EnableTvDbUpdates`, `chkEnableTmdbUpdates` and `chkEnableFanartUpdates` checks here?

Reply:
None of the three had UI elements, they must have been old settings.